app/services/testdata/RegisteredStatusGenerator.scala

```diff
@@ -39,9 +39,9 @@ trait RegisteredStatusGenerator extends BaseGenerator {
 
   def generate(generationId: Int, generatorConfig: GeneratorConfig)(implicit hc: HeaderCarrier, rh: RequestHeader) = {
     val firstName = generatorConfig.firstName.getOrElse(Random.getFirstname(generationId))
-    val lastName = generatorConfig.lastName.getOrElse(Random.getLastname(generationId + 4000))
+    val lastName = generatorConfig.lastName.getOrElse(Random.getLastname(generationId))
     val preferredName = generatorConfig.preferredName.getOrElse(s"Pref$firstName")
-    val email = s"${generatorConfig.emailPrefix}${generationId + 4000}@mailinator.com"
+    val email = s"${generatorConfig.emailPrefix.getOrElse("tesf" + Random.number())}a${generationId}@mailinator.com"
```

Comment:
Should this be 'test'? What's with all the random characters?

Should we use 'fstest' and a '-' instead  of the 'a' ?